pandas-stubs/_typing.pyi

```diff
@@ -318,7 +318,7 @@ MergeHow: TypeAlias = Union[JoinHow, Literal["cross"]]
 JsonFrameOrient: TypeAlias = Literal[
     "split", "records", "index", "columns", "values", "table"
 ]
-JsonSeriesOrient: TypeAlias = Literal["split", "records", "index"]
+JsonSeriesOrient: TypeAlias = Literal["split", "records", "index", "table"]
```

Comment:
There seem to be a few more orientation values https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.to_json.html that can be added.

Ideally, we would also have a test for this.